.travis.yml

```diff
@@ -1,9 +1,10 @@
 jdk:
   - oraclejdk8
+  - oraclejdk9
 language: scala
 sudo: false
 script:
    - sbt ^test ^scripted
    - cd examples
-   - for d in */ ; do cd "$d" &&  sbt test && cd ../ ; done
+   - for d in */ ; do cd "$d" && $(if [ -f "./project/build.properties" ]; then echo "sbt test" ; else echo "sbt -sbt-version 0.13.17 test" ; fi) && cd ../ ; done
```

Comment:
This seems unintentional: echoing instead of running? Wouldn't it better if we always put a `project/build.properties`